new/src/App.tsx

```diff
@@ -13,6 +14,7 @@ const App = (): JSX.Element => {
       <About />
       <Portfolio />
       <Contact />
+      <DiscordSection />
```

Comment:
I think Discord section and contact are the same? There is no separate section for contact.

On a separate note, I think the current Figma section is missing other means of contact. We may use the current `<Contact />` component to fill in the details as a separate section, or put the contacts in the same section as the discord link. Lmk what you intend to do.